CiscoVM/CiscoVM-1.0.0/property.conf

```diff
@@ -9,50 +9,78 @@
   ],
   "properties": [
     {
-      "tag": "connect_ciscovm_exported",
-      "label": "Cisco VM Exported",
-      "description": "Cisco Vulnerability Management Exported",
+      "tag": "connect_ciscovm_exported_state",
+      "label": "Cisco VM Exported State",
+      "description": "State of an endpoint export",
       "type": "string",
       "group": "connect_ciscovm",
+      "resolvable": false
+    },
+    {
+      "tag": "connect_ciscovm_exported_hash",
+      "label": "Cisco VM Exported Hash",
+      "description": "Hash of payload to avoid redundant exports",
+      "type": "string",
+      "group": "connect_ciscovm",
+      "resolvable": false
+    }
+  ],
+  "action_groups": [
+    {
+      "name": "connect_ciscovm_actions",
+      "label": "Cisco VM"
+    }
+  ],
+  "actions": [
+    {
+      "name": "connect_ciscovm_export",
+      "label": "Cisco VM Export",
+      "group": "connect_ciscovm_actions",
+      "description": "Export assets to Cisco VM",
+      "ip_required": false,
```

Comment:
I'm wondering. Why not required?

Reply:
Based on our mapping rules, mac address is enough, so missing ip address shouldn't block the extraction.